index.html

```diff
@@ -283,6 +283,28 @@ <h3>
     identified by the token "compute-pressure".
     Its [=policy-controlled feature/default allowlist=] is `["self"]`.
   </p>
+  <p>
+    Workers (dedicated and shared) adhere to the permission policy set by their
+    owning document(s).
+  </p>
+  <p>
+    Shared workers often have multiple owning documents as they can be obtained
+    by other documents with the [=same origin=].
+
+    In this case, all owning documents have to have the permission policy allowed
```

Comment:
Is "have the permission policy allowed" a common way of phrasing this? It sounds a bit confusing.

Reply:
Yeah, I tried changing it a few times, but didn't find a good wording :-) Maybe @anssiko has some good suggestions

Reply:
How about just using [allowed to use](https://html.spec.whatwg.org/multipage/iframe-embed-object.html#allowed-to-use) like in `observe()`?

_"[...] all owning documents must be allowed to use the policy-controlled feature defined by this specification"_, for example.

Reply:
Sure, done